### ISSUE TITLE
While VibranceGUI is active I can't change the refresh rate in Control Panel

### ISSUE THREAD

**Issue by Ghosty141** [juvlarN/vibranceGUI#66]
When I start CS:GO my monitors refresh rate (Acer Predator X34P, 120hz GSync) is set to 120 Hz with GSync On. After quitting the game it gets reset to 60hz or 50hz and whenever I want to change it back to 120hz  (after hitting "Apply"), the screen flickers and it's back to 60hz/50hz.

Specs: 
Win10 Home, 1709 (16299.371)
GTX 1070 with 391.35 drivers

**Comment by juvlarN** [comment]
Are you sure you have 120 Hz enabled when in Windows? 

VibranceGUI retrieves the Windows desktop settings (resolution, refresh rate) on program startup and will make sure that this exact desktop settings are not being changed. This is due to the fact that you can select the "Change Resolution when Ingame" checkbox, which will make vibranceGUI change your resolution to whatever you set in there. And when you tab out of the game, the settings that were recorded on program startup will be loaded. Also, every time when you switch between applications and the resolution does not match the Windows resolution that were present on program startup, the resolution will change again - that's what you call "flickering" I guess. You change the resolution manually, vibranceGUI detects that the resolution has changed and is not the same as on program start up, then changes it again and this will cause the flickering effect.

**Comment by Ghosty141** [comment]
Yes, the 120Hz are enabled in the Windows desktop settings, as welll as in the nvidia control panel.

![rundll32_2018-04-22_19-46-26](https://user-images.githubusercontent.com/17564201/39098100-efe1698e-4665-11e8-9920-6733ae657294.png)

The problem only started when I got my new monitor, my previous monitor worked fine. The only "big" difference between them is that the Acer Predator x34p only supports 100hz natively and 120hz with its overclock feature. The weirdest part about the issue is that it gets switched back to 60hz while it was never set to 60hz/50hz (apart from 10 minutes while setting up the monitor since it was hooked up with HDMI rather than DP).

**Comment by camjac251** [comment]
Thank you for this fix @juvlarN
